server/index.js

```diff
@@ -1,9 +1,60 @@
 const express = require('express');
 const path = require('path');
-const port = process.env.PORT || 3000;
+const bodyParser = require('body-parser');
+const db = require('../database/index.js');
 
 const app = express();
 
+app.use(bodyParser.json());
 app.use(express.static(path.join(__dirname, '../public')));
 
-app.listen(port, () => console.log(`Server listening on port ${port}`));
\ No newline at end of file
+
+//GET business page reviews (initial resaturant page = random, or search by id)
+app.get('/business/:id/reviews', (req, res) => {
+    db.businessPageReviewsLoading(req.params.id, (err, result) => {
+        if (err) {
+            console.log("Error: fetch review from db - business id");
```

Comment:
You may also want to send some status report to the client so they know there was a server side error